src/account-settings/AccountSettingsPage.jsx

```diff
@@ -115,10 +116,10 @@ class AccountSettingsPage extends React.Component {
 
     return (
       <div>
+        {this.renderBanner()}
```

Comment:
I think you can swap this now with:
```suggestion
        <BetaLanguageBanner />
```